pyston/serializer.py

```diff
@@ -336,6 +340,13 @@ def _get_m2m_fields(self, obj):
                 out[mf.name] = mf
         return out
 
+    def _get_reverse_fields(self, obj):
+        out = {}
```

Comment:
how about a better variable name, like reverse_fields?

Reply:
yes you are right. I used same naming like other methods. I will fix it